packages/browser/src/browser/index.ts

```diff
@@ -79,11 +79,9 @@ export interface AnalyticsBrowserSettings extends AnalyticsSettings {
 
 export function loadLegacySettings(
```

Comment:
I believe this a breaking change. I have seen some users actually use this function in their own code.